wire-ios-system/Source/Logging/LogConvertible.swift

```diff
@@ -16,12 +16,16 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-extension WireLogger {
-    public static func addDatadog(_ datadog: LoggerProtocol) {
-        if let aggregatedLogger = Self.provider as? AggregatedLogger {
-            aggregatedLogger.addLogger(datadog)
-        } else {
-            Self.provider = datadog
-        }
```

Comment:
This tangled code was the reason to do the cleanup as `AggregatedLogger` always existed with internally given instances.